include/igl/writeOBJ.cpp

```diff
@@ -118,6 +118,43 @@ IGL_INLINE bool igl::writeOBJ(
   return true;
 }
 
+template <typename DerivedV, typename T>
+IGL_INLINE bool igl::writeOBJ(
+  const std::string &str,
+  const Eigen::MatrixBase<DerivedV>& V,
+  const std::vector<std::vector<T> >& F)
+{
+  using namespace std;
+  using namespace Eigen;
+  assert(V.cols() == 3 && "V should have 3 columns");
+  ofstream s(str);
+  if(!s.is_open())
+  {
+    fprintf(stderr,"IOError: writeOBJ() could not open %s\n",str.c_str());
+    return false;
+  }
+  s<<V.format(IOFormat(FullPrecision,DontAlignCols," ","\n","v ","","","\n"));
+  
+  for(const auto& face : F) {
+    int face_size = face.size();
+    
+    assert(face_size != 0);
+    
+    if(face_size == 2) {
```

Comment:
Btw, the braces in your commit do not respect the [style guideline](https://libigl.github.io/style-guidelines/). Can you please fix that? For this specific line, I would use a trigraph instead: `s << (face_size == 2 ? "l" : "f")`